util/httputil/client_test.go

```diff
@@ -272,6 +302,32 @@ func TestBearerAuthRoundTripper(t *testing.T) {
 	bearerAuthRoundTripperShouldNotModifyExistingAuthorization.RoundTrip(request)
 }
 
+func TestBearerAuthFileRoundTripper(t *testing.T) {
+	const (
+		newBearerToken = "goodbyeandthankyouforthefish"
+	)
+
+	fakeRoundTripper := testutil.NewRoundTripCheckRequest(func(req *http.Request) {
+		bearer := req.Header.Get("Authorization")
+		if bearer != ExpectedBearer {
+			t.Errorf("The expected Bearer Authorization (%s) differs from the obtained Bearer Authorization (%s)",
+				ExpectedBearer, bearer)
+		}
+	}, nil, nil)
+
+	//Normal flow
```

Comment:
space after `//` and should end with `.`.

Same below.

Reply:
fixed. 
btw these were some comment from a someone elses commit, so no red points for me :)

Reply:
@Gouthamve ^

Reply:
@krasi-georgiev it was only fixed in the comments below, not these